udata/commands/fixtures.py

```diff
@@ -1,49 +1,32 @@
 # -*- coding: utf-8 -*-
-
 from __future__ import unicode_literals
 
-from datetime import datetime
 import logging
 
 from udata.commands import manager
-from udata.core.dataset.factories import VisibleDatasetFactory, LicenseFactory
+from udata.core.dataset.factories import VisibleDatasetFactory
 from udata.core.discussions.factories import DiscussionFactory
-from udata.core.organization.factories import OrganizationFactory, TeamFactory
-from udata.core.reuse.factories import VisibleReuseFactory
-from udata.core.user.factories import AdminFactory
+from udata.core.organization.factories import OrganizationFactory
+from udata.core.organization.models import Member
+from udata.core.reuse.factories import ReuseFactory
+from udata.core.user.factories import UserFactory
 
 log = logging.getLogger(__name__)
 
 
-def generate_datasets(count, organization=None):
-    for _ in range(0, count):
-        dataset = VisibleDatasetFactory(organization=organization)
-        DiscussionFactory(subject=dataset)
-
-
-def generate_reuses(count, user=None):
-    VisibleReuseFactory.create_batch(count, owner=user)
-
-
-def generate_licenses(count):
-    for _ in range(0, count):
-        LicenseFactory()
-
-
 @manager.command
-def generate_fixtures():
+def generate_fixtures(datasets=5, reuses=1):
     '''Build sample fixture data (users, datasets and reuses).'''
-    user = AdminFactory(email='user@udata', password='password',
-                        confirmed_at=datetime.now())
-    log.info('Generated admin user "user@udata" with password "password".')
+    user = UserFactory()
+    log.info('Generated user "{user.email}".'.format(user=user))
```

Comment:
No more password display? I guess this is because the "right way" is now to set your own password when prompted to create a super user. But what about a non-interactive install? Is it not interesting to display the password in this case (docker install)?